src/test/java/io/in_toto/keys/RSAKeyTest.java

```diff
@@ -1,19 +1,28 @@
 package io.in_toto.keys;
 
-import java.io.IOException;
+import io.in_toto.keys.RSAKey;
 
-import java.lang.System;
+import java.io.File;
+import java.io.IOException;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import org.junit.jupiter.api.Test;
 
-import io.in_toto.keys.RSAKey;
+//import static org.hamcrest.MatcherAssert;
+//import static org.hamcrest.beans.SamePropertyValuesAs;
```

Comment:
what is this for?

Reply:
I tried this to test the equality for getPublic, getPrivate, and getSigner methods, but was not helpful. Now removed.